src/ModuleTestLite.cs

```diff
@@ -341,7 +341,12 @@ private void triggerFailure(int type)
 					type = (int)failureTypes.PERFLOSS;
 				}
 			}
-			failure_du += failureData[type];
+            // No failure data until the craft has launched
+            if (runTime > clampTime)
+            {
+                // Roll off data above rated run time, to a minimum of 10%
```

Comment:
I don't agree with the motivation behind this change.  IMO run-to-destruction is a valid engineering test; note that for the vast majority of rocket engine models IRL, engines are ground-tested to lives considerably longer than our 'rated burn times'.  See e.g. [Henry Spencer](https://yarchive.net/space/rocket/engine_reusability.html).

Moreover, tooling reasons mean that mostly test flights of new engines in-game are likely to use the 'production' stage (tankage, and avionics!) — an overburning test flight is not a trivial thing to arrange.